### PR TITLE
add some missing Kinds to libclang python bindings

### PR DESCRIPTION
**Pull request by 16bit-ykiko**
add some Kinds existing in Index.h but missing in cindex.py.

### CODE REVIEW

**Comment by 16bit-ykiko** [comment]
Pinging @Endilll and @linux4life798 for review.

**Review by AaronBallman — COMMENTED** [review]
Code changes LGTM (we don't require test coverage for bindings, so that's fine), but can you add a release note to `clang/docs/ReleaseNotes.rst` under `Python Binding Changes` so others know about the improvement?

**Comment by 16bit-ykiko** [comment]
Sure, I have added the releasenote.

**Review by AaronBallman — APPROVED** [review]
LGTM!

File: clang/docs/ReleaseNotes.rst [hunk 1]
```diff
@@ -544,6 +544,8 @@ Python Binding Changes
 ----------------------
 
 - Exposed `CXRewriter` API as `class Rewriter`.
+- Add some missing kinds from Index.h (CursorKind: 149-156, 272-320, 420-437. 
+  TemplateArgumentKind: 5-9. TypeKind: 161-175 and 178).
```
Comment (AaronBallman) [inline clang/docs/ReleaseNotes.rst:548]:
```suggestion
- Add some missing kinds from Index.h (CursorKind: 149-156, 272-320, 420-437.
  TemplateArgumentKind: 5-9. TypeKind: 161-175 and 178).
```

**Review by AaronBallman — COMMENTED** [review]
Nit, I'll fix when landing.

**Review by Endilll — COMMENTED** [review]
This looks good.
Sorry for the late reply.

**Comment by schenker** [comment]
Thanks for the fix. Will these changes find their way into an 18.x release?

**Comment by Endilll** [comment]
> Will these changes find their way into a 18.x release?

Not without someone stepping up and getting them through backporting process.

**Pull request merged**